### PR TITLE
notification module fixes#16

### PR DESCRIPTION
**Pull request by Jayashree-panda**

### CODE REVIEW

File: src/lib/modules/notification/notify.go [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+	icon        string
+	urgency     string
+	time        int
+}
+
+//NotificationManage sends notifications to the user based on different params passed
+//All the methods in this interface based on different params
+type NotificationManage interface {
+	//Notify1 with title and description
+	Notify1()
```
Comment (ankitjena) [inline src/lib/modules/notification/notify.go]: @Jayashree-panda you can give better names to the functions

File: src/lib/modules/notification/notify.go [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+	cmd := exec.Command("notify-send", N.title, N.description, "-u", N.urgency)
+	err := cmd.Run()
+	if err != nil {
+		fmt.Println(err)
+	}
+}
+
+// Notify5 executes the command to notify with title, description, icon and time
+func (N *NotificationData) Notify5() {
+	cmd := exec.Command("notify-send", N.title, N.description, "-i", "/home/jayashree/go/src/github.com/zairza-cetb/bench-routes/assets/jj.png", "-t", strconv.Itoa(N.time))
```
Comment (ankitjena) [inline src/lib/modules/notification/notify.go]: This is hardcoded to your system, it should be generic

**Comment by Harkishen-Singh** [comment]
@Jayashree-panda  also, commit along with yr sign-off.

`git commit -s -m "<commit message>"`

File: src/lib/modules/notification/notify.go (hunk 2 above)
Comment (Harkishen-Singh) [reply src/lib/modules/notification/notify.go]: @Jayashree-panda better if you use the command name declared as a constant at the top of the file, like the one done in flood ping.

**Comment by ankitjena** [comment]
DCO fails due to commit signature, merging this.

**Comment by Harkishen-Singh** [comment]
@ankitjena, pls squash the commits

**Comment by ankitjena** [comment]
@Harkishen-Singh Yes squashed them!

**Pull request merged**